cypress/integration/home.spec.ts

```diff
@@ -1,3 +1,40 @@
+it('should show correct page title', () => {
+  cy.visit('/')
+
+  cy.title().should(
+    'equal',
+    'Homepage | Octoclairvoyant: Compare GitHub changelogs across multiple releases'
+  )
+})
+
+it('should display h1 heading with correct text', () => {
+  cy.visit('/')
+
+  cy.findByRole('heading', { level: 1, name: 'Octoclairvoyant' }).should(
+    'exist'
+  )
+})
+
+it('should display h2 heading with correct text', () => {
+  cy.visit('/')
+
+  cy.findByRole('heading', {
+    level: 2,
+    name: 'Compare GitHub changelogs across multiple releases',
+  }).should('exist')
+})
+
+it('should display h3 heading with correct text', () => {
+  cy.visit('/')
+
+  cy.findByRole('heading', {
+    level: 3,
+    name: 'Features',
+  }).should('exist')
+})
+
+// TODO: Write test that checks for image being present (alt text is already checked via some other linter).
```

Comment:
You should use `ByAltText` for getting images: https://testing-library.com/docs/queries/byalttext

Once you have it, you can check that the `src` attribute has the desired value.